package.json

```diff
@@ -14,15 +14,15 @@
     }
   },
   "scripts": {
-    "prepare": "tshy",
+    "build": "pnpm run tshy && pnpm run format",
```

Comment:
It should remain in "prepare" but you can also add a build task if you want, e.g.

```
    "build": "tshy && pnpm run format",
    "prepare": "tshy",
```

Reply:
Is "prepare" a standard name?  I don't really care what things are called. I was just annoyed that running tshy resulted in git diffs :/